magento/adobe-stock-integration
# 1641: The loop symbol overlaps the text in Used in Products filter [Device mod]

### Description (*)

This PR will fix the overlapping issue between the loop symbol and the text in Used in Products filter.
### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration https://github.com/magento/adobe-stock-integration/issues/1641: The loop symbol overlaps the text in Used in Products filter [Device mod]

### Manual testing scenarios (*)


1. Go to **Content - Media Gallery**,
2. In your web browser, turn on _Toggle Device Toolbar_ and select _iPad_ device for example
3. Click **Filters**
4. Click **Used in Products** dropdown
5. The icon and the placeholder text should not be overlapping.
